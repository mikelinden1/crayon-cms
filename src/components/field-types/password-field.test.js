import PasswordField from './password-field';

const props = {
    name: 'test',
    value: 'value',
    onChange: () => {},
    disabled: false
};

it('PasswordField matches snapshot', () => {
    const wrapper = shallow(<PasswordField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
