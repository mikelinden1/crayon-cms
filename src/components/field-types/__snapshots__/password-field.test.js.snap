// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordField matches snapshot 1`] = `
<div
  className="row"
>
  <div
    className="col-md-9"
  >
    <input
      className="form-control"
      disabled={false}
      name="test"
      onChange={[Function]}
      type="password"
      value="value"
    />
  </div>
  <div
    className="col-md-3"
  >
    <div
      className="custom-control custom-checkbox"
      style={
        Object {
          "marginTop": "10px",
        }
      }
    >
      <input
        checked={false}
        className="custom-control-input"
        disabled={false}
        id="show-password"
        name="show-password"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="custom-control-label"
        htmlFor="show-password"
      >
        Show Password
      </label>
    </div>
  </div>
</div>
`;
